tests/parm/input_global_nest.nml.IN

```diff
@@ -0,0 +1,401 @@
+  res_latlon_dynamics = @[RES_LATLON_DYNAMICS]
+  nested = @[NESTED]
+  twowaynest = @[TWOWAYNEST02]
+  nestupdate = 7
+/
+
+&external_ic_nml
+  filtered_terrain = .true.
+  levp = 128
+  gfs_dwinds = .true.
+  checker_tr = .false.
+  nt_checker = 0
+  
+/
+
+&surf_map_nml
+  cd2 = -1
+  cd4 = 0.15
+  max_slope = 0.15
+  n_del2_strong = @[N_DEL2_STRONG]
+  n_del2_weak = @[N_DEL2_WEAK_NEST02]
+  n_del4 = @[N_DEL4]
+  peak_fac = @[PEAK_FAC]
+  zero_ocean = @[ZERO_OCEAN]
+/
+
+&gfs_physics_nml
+  fhzero       = @[FHZERO]
+  h2o_phys     = @[H2O_PHYS]
+  ldiag3d      = @[LDIAG3D]
+  qdiag3d      = @[QDIAG3D]
+  lightning_threat = @[LIGHTNING_THREAT]
+  print_diff_pgr = @[PRINT_DIFF_PGR]
+  fhcyc        = @[FHCYC]
+  use_ufo      = .true.
+  pre_rad      = .false.
+  imp_physics  = @[IMP_PHYSICS]
+  iovr         = @[IOVR]
+  lcnorm       = @[LCNORM]
+  ltaerosol    = .true.
+  lradar       = .true.
+  ttendlim     = -999
+  dt_inner     = @[DT_INNER]
+  sedi_semi    = @[SEDI_SEMI]
+  decfl        = @[DECFL]
+  oz_phys      = @[OZ_PHYS_NEW]
+  oz_phys_2015 = @[OZ_PHYS_OLD]
+  lsoil_lsm    = @[LSOIL_LSM]
+  do_ca = @[DO_CA]
+  do_mynnedmf  = @[DO_MYNNEDMF]
+  do_mynnsfclay = @[DO_MYNNSFCLAY]
+  icloud_bl    = 1
+  bl_mynn_edmf = 1
+  bl_mynn_tkeadvect = @[BL_MYNN_TKEADVECT]
+  bl_mynn_edmf_mom = 1
+  do_ugwp      = @[DO_UGWP]
+  do_tofd      = @[DO_TOFD]
+  gwd_opt      = @[GWD_OPT]
+  do_ugwp_v0   = @[DO_UGWP_V0]
+  do_ugwp_v1   = @[DO_UGWP_V1]
+  do_ugwp_v0_orog_only = @[DO_UGWP_V0_OROG_ONLY]
+  do_ugwp_v0_nst_only  = @[DO_UGWP_V0_NST_ONLY]
+  do_gsl_drag_ls_bl    = @[DO_GSL_DRAG_LS_BL]
+  do_gsl_drag_ss       = @[DO_GSL_DRAG_SS]
+  do_gsl_drag_tofd     = @[DO_GSL_DRAG_TOFD]
+  do_ugwp_v1_orog_only = @[DO_UGWP_V1_OROG_ONLY]
+  do_ugwp_v1_w_gsldrag = @[DO_UGWP_V1_W_GSLDRAG]
+  min_lakeice  = @[MIN_LAKEICE]
+  min_seaice   = @[MIN_SEAICE]
+  use_cice_alb = @[USE_CICE_ALB]
+  pdfcld       = .false.
+  fhswr        = @[FHSWR]
+  fhlwr        = @[FHLWR]
+  ialb         = @[IALB]
+  iems         = @[IEMS]
+  iaer         = @[IAER]
+  icliq_sw     = @[ICLIQ_SW]
+  ico2         = 2
+  isubc_sw     = 2
+  isubc_lw     = 2
+  isol         = 2
+  lwhtr        = .true.
+  swhtr        = .true.
+  cnvgwd       = @[CNVGWD_NEST02]
+  shal_cnv     = @[SHAL_CNV]
+  cal_pre      = .false.
+  redrag       = .true.
+  dspheat      = .true.
+  hybedmf      = @[HYBEDMF]
+  satmedmf     = @[SATMEDMF]
+  isatmedmf    = @[ISATMEDMF]
+  lheatstrg    = @[LHEATSTRG]
+  lseaspray    = @[LSEASPRAY]
+  random_clds  = @[RANDOM_CLDS]
+  trans_trac   = .true.
+  cnvcld       = @[CNVCLD]
+  imfshalcnv   = @[IMFSHALCNV_NEST02]
+  imfdeepcnv   = @[IMFDEEPCNV_NEST02]
+  do_deep      = @[DO_DEEP_NEST02]
+  ! NOTE: progsigma should be .true. for the global_nest_v1 suite.
+  ! The code turns it off automatically at low resolutions (nominally, less than C768 resolution).
+  ! That means it would turn off for the regression tests (C96 globe, C384 resolution nest).
```

Comment:
@SamuelTrahanNOAA just a remark regarding your comment here. progsigma turns off less than C384 resolution (not C768 as indicated in this comment).

Reply:
That's an unwise design. If it is turned off in the globe, and on in the nest, then the nest is advecting 0 at the boundary.

Reply:
I've updated the comments accordingly. (The same comment can be found in both namelists and the gnv1_nested.)